python-test.py

```diff
@@ -1,11 +1,11 @@
 print ("This is a tttsting code to check error catching")
 
-def simple_fn(range):
-    if range > 0:
+def simple_fn(range, seed):
+    if range >= 0 and seed == 1:
         for i in range(range):
             print("Hi!")
```

Comment:
Rename parameter 'range' to 'num_range' to avoid shadowing the built-in function